src/plots/xyPlot.ts

```diff
@@ -53,10 +152,6 @@ export class XYPlot<X, Y> extends Plot {
       this._updateYExtentsAndAutodomain();
     }
 
-    if (xScale != null) {
-      xScale.onUpdate(this._adjustYDomainOnChangeFromXCallback);
```

Comment:
Redundant. `this.bindProperty()` -> `_bind()`-> `_installScaleForKey()` -> `scale.onUpdate(adjustDomainCallback)`